tests/foreman/ui/test_docker.py

```diff
@@ -1279,6 +1280,34 @@ def test_positive_delete(self):
                     self.assertIsNone(
                         self.container.search(compute_resource.name, name))
 
+    # BZ 1266842 is a private bug, so remove that stub once it is fixed
+    @stubbed
```

Comment:
Maybe it's better to specify a reason here like we did it in [#3277](https://github.com/SatelliteQE/robottelo/pull/3277/files#diff-7348c3a9bfd682d700bc0c8540410544R324) and [#3274](https://github.com/SatelliteQE/robottelo/pull/3274/files#diff-aa8831ff56496154fb9a4a2754c259fdR543)?